newIDE/app/src/BrowserApp.js

```diff
@@ -41,7 +41,10 @@ export const create = (authentification: Authentification) => {
       eventsFunctionsExtensionWriter={null}
       eventsFunctionsExtensionOpener={null}
     >
-      {({ i18n, eventsFunctionsExtensionsState }) => (
+      {({
```

Comment:
nitpicking: can you revert the changes to these lines where nothing was actually changed? Will speed up review for me :)